packages/twenty-server/src/engine/metadata-modules/agent/dtos/update-agent.input.ts

```diff
@@ -39,8 +50,18 @@ export class UpdateAgentInput {
   @Field(() => String)
   modelId?: ModelId;
 
+  @IsString()
+  @IsOptional()
+  @Field(() => UUIDScalarType, { nullable: true })
+  roleId?: string;
+
   @IsObject()
   @IsOptional()
   @Field(() => GraphQLJSON, { nullable: true })
   responseFormat?: object;
+
+  @IsBoolean()
+  @IsOptional()
+  @Field({ nullable: true })
+  isCustom?: boolean;
```

Comment:
I shouldn't be able to update if an agent isCustom or not. Standard agent are only created through code and cannot be created by an external user